roles/jd-server/src/main.rs

```diff
@@ -108,163 +103,5 @@ async fn main() {
         }
     };
 
```

Comment:
I think it would be helpful to keep this sort of daemon awareness logging.
```suggestion
info!("Jds INITIALIZING with config: {:?}", &args.config_path);
```